### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by hjia095**

### CODE REVIEW

**Comment by bjur781** [comment]
Looks good! just a couple of changes:
- can it be referenced that this is a softeng 310 project somewhere? and maybe the teams that have contributed to it
- and I believe that python requires version 3.10, not just python 3

File: README.md [hunk 1]
```diff
@@ -1,20 +1,90 @@
...
+<p>
+The objective of Asteroids is to destroy asteroids. The player controls a triangular ship that can rotate left and right, fire shots straight forward, and thrust forward. Scores are earned when hits asterorids.
+</p>
+
+![image](https://user-images.githubusercontent.com/79783194/192207603-3ce24e4f-6659-450a-8491-88084c298a04.png)
+
+## Contribution
+PyArcade is developed by SOFTENG 310 Team 2 for Assignment 1 and take over by SOFTENG 310 Team 5 for Assignment 2
+
+Detailed contribution please reference to the [Wiki](https://github.com/Biscuitmunch/310A1Team2/wiki) pages [list of contribution].(https://github.com/Biscuitmunch/310A1Team2/wiki/List-of-Contributions)
```
Comment (bjur781) [inline README.md:84]: the link to the list of contributions page isn't formatting nicely, I'm not sure if you can reference specific pages of the wiki, so maybe just leave it as the wiki?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,90 @@
...
+## Menu
+  - [🕹️ GAMES](#%EF%B8%8F-games)
+    - [🐍 Snake](#-snake)
+    - [🧱 Breakout](#-breakout)
+    - [🏓 Pong](#-pong)
+    - [👾 Invader](#-invader)
+    - [🚀 Asteroids](#-asteroids)
 
 ## Installation
+To run the arcade, you will need to have python3 installed, as well as pygame.
```
Comment (bjur781) [inline README.md:22]: could this be changed to python3.10 as well to make it super super clear?

**Review by bjur781 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hjia095) [reply README.md:84]: OK, I will make this only reference to the list of contribution

File: README.md (hunk 2 above)
Comment (hjia095) [reply README.md:22]: will do

**Pull request merged**